platforms/guardoni/docs/intro.md

```diff
@@ -18,7 +18,7 @@ Having trouble? We’d like to help!
 
 - Check the **[Github](https://github.com/tracking-exposed)** repo, report bugs or contribute. ❤️
 
-- Join the community in Tracking Exposed **[Slack](trackingexposed.slack.com)**.
+- Join the community in Tracking Exposed **[Slack](https://trackingexposed.slack.com)**.
```

Comment:
sorry, slack isn't supposed to be publicly accessible. we don't have a place (yet) for support. perhaps, if we move all our channels as private, we can consider this.